frontend/src/modules/Shares/components/ShareBoxListItem.js

```diff
@@ -0,0 +1,95 @@
+import { Box, Button, Card, Grid, Typography } from '@mui/material';
+import PropTypes from 'prop-types';
+import { Link as RouterLink } from 'react-router-dom';
+import { ShareStatus, useCardStyle } from 'design';
+
+export const ShareBoxListItem = ({ share }) => {
+  const classes = useCardStyle();
+
+  return (
+    <Card
+      key={share.shareUri}
+      className={classes.card}
+      sx={{
+        mt: 2
+      }}
+    >
+      <Grid container spacing={1} alignItems="center">
+        <Grid item justifyContent="center" md={1} xs={1}>
+          <Box
+            sx={{
+              pt: 2,
+              pb: 2,
+              px: 3
+            }}
+          >
+            <ShareStatus status={share.status} />
+          </Box>
+        </Grid>
+        <Grid item justifyContent="flex-end" md={5} xs={5}>
+          <Box
+            sx={{
+              pt: 2,
+              pb: 2,
+              px: 3
+            }}
+          >
+            <Typography color="textPrimary" variant="body1">
+              {`Request owner [principal]`}
```

Comment:
Minor - we are using `{TEXT}` in Typography even when text is static in this file